src/pages/accounting/AccountCard.tsx

```diff
@@ -40,81 +40,60 @@ interface ChequebookBalance {
-                  </Typography>
-                  <Typography component="p" variant="h5">
-                    {fromBZZbaseUnit(props.chequebookBalance.availableBalance)}
-                  </Typography>
-                </Grid>
-                <Grid item>
-                  <Typography component="h2" variant="h6" color="primary" gutterBottom>
-                    Total Sent / Received (BZZ)
-                  </Typography>
-                  <Typography component="div" variant="h5">
-                    <span style={{ marginRight: '7px' }}>
-                      {fromBZZbaseUnit(props.settlements?.totalsent || 0)} /{' '}
-                      {fromBZZbaseUnit(props.settlements?.totalreceived || 0)}
-                    </span>
-                    <span
-                      style={{
-                        color:
-                          props.settlements && props.settlements?.totalsent > props.settlements?.totalreceived
-                            ? '#c9201f'
-                            : '#32c48d',
-                      }}
-                    >
-                      (
-                      {fromBZZbaseUnit(
-                        (props.settlements && props.settlements?.totalsent - props.settlements?.totalreceived) || 0,
-                      )}
-                      )
-                    </span>
-                  </Typography>
-                </Grid>
-              </Grid>
-            </CardContent>
-          </div>
-        ) : (
-          <div className={classes.details}>
-            <Skeleton width={180} height={110} animation="wave" style={{ marginLeft: '12px', marginRight: '12px' }} />
-            <Skeleton width={180} height={110} animation="wave" style={{ marginLeft: '12px', marginRight: '12px' }} />
-            <Skeleton width={180} height={110} animation="wave" style={{ marginLeft: '12px', marginRight: '12px' }} />
+        {!isLoading && (
+          <CardContent className={classes.gridContainer}>
+            <div>
+              <Typography component="h2" variant="h6" color="primary" gutterBottom>
+                Total Balance
+              </Typography>
+              <Typography component="p" variant="h5">
+                {fromBZZbaseUnit(chequebookBalance?.totalBalance || 0).toFixed(7)} BZZ
+              </Typography>
+            </div>
+            <div>
+              <Typography component="h2" variant="h6" color="primary" gutterBottom>
+                Available Uncommitted Balance
+              </Typography>
+              <Typography component="p" variant="h5">
+                {fromBZZbaseUnit(chequebookBalance?.availableBalance || 0).toFixed(7)} BZZ
+              </Typography>
+            </div>
+            <div>
+              <Typography component="h2" variant="h6" color="primary" gutterBottom>
+                Total Sent / Received
+              </Typography>
+              <Typography component="div" variant="h5">
+                <span style={{ marginRight: '7px' }}>
```

Comment:
`7px` stands out to me, since `theme` has spacings, also it is not an increment of 4 or 8 units (not sure if we follow that here).

Reply:
Yeah I am removing it all together as the margin is not needed at all now thanks to the `columnGap` spacing.